bin/git-fork

```diff
@@ -17,9 +17,13 @@ read -r user
 # personal access token
 # config name is github-personal-access-token '_' is not allowed in git config
 
-github_personal_access_token=$(git config git-extras.github-personal-access-token)
+if test -z "${GITHUB_TOKEN}"; then
```

Comment:
Better to use `test -n`, and put the highest priority branch (using GITHUB_TOKEN) as the first one. This way is more readable.